src/MongoDB.Driver/Linq/Linq3Implementation/Reflection/StringMemberInfo.cs

```diff
@@ -124,44 +126,45 @@ static StringMethod()
-        public static MethodInfo IndexOfAny => __indexOfAny;
-        public static MethodInfo IndexOfAnyWithStartIndex => __indexOfAnyWithStartIndex;
-        public static MethodInfo IndexOfAnyWithStartIndexAndCount => __indexOfAnyWithStartIndexAndCount;
-        public static MethodInfo IndexOfWithChar => __indexOfWithChar;
-        public static MethodInfo IndexOfWithCharAndStartIndex => __indexOfWithCharAndStartIndex;
-        public static MethodInfo IndexOfWithCharAndStartIndexAndCount => __indexOfWithCharAndStartIndexAndCount;
-        public static MethodInfo IndexOfWithString => __indexOfWithString;
-        public static MethodInfo IndexOfWithStringAndStartIndex => __indexOfWithStringAndStartIndex;
-        public static MethodInfo IndexOfWithStringAndStartIndexAndCount => __indexOfWithStringAndStartIndexAndCount;
-        public static MethodInfo IndexOfWithStringAndComparisonType => __indexOfWithStringAndComparisonType;
-        public static MethodInfo IndexOfWithStringAndStartIndexAndComparisonType => __indexOfWithStringAndStartIndexAndComparisonType;
-        public static MethodInfo IndexOfWithStringAndStartIndexAndCountAndComparisonType => __indexOfWithStringAndStartIndexAndCountAndComparisonType;
-        public static MethodInfo IsNullOrEmpty => __isNullOrEmpty;
-        public static MethodInfo SplitWithChars => __splitWithChars;
-        public static MethodInfo SplitWithCharsAndCount => __splitWithCharsAndCount;
-        public static MethodInfo SplitWithCharsAndCountAndOptions => __splitWithCharsAndCountAndOptions;
-        public static MethodInfo SplitWithCharsAndOptions => __splitWithCharsAndOptions;
-        public static MethodInfo SplitWithStringsAndCountAndOptions => __splitWithStringsAndCountAndOptions;
-        public static MethodInfo SplitWithStringsAndOptions => __splitWithStringsAndOptions;
-        public static MethodInfo StartsWith => __startsWith;
-        public static MethodInfo StartsWithWithComparisonType => __startsWithWithComparisonType;
-        public static MethodInfo StartsWithWithIgnoreCaseAndCulture => __startsWithWithIgnoreCaseAndCulture;
-        public static MethodInfo Substring => __substring;
-        public static MethodInfo SubstringWithLength => __substringWithLength;
-        public static MethodInfo ToLower => __toLower;
-        public static MethodInfo ToLowerInvariant => __toLowerInvariant;
-        public static MethodInfo ToLowerWithCulture => __toLowerWithCulture;
-        public static MethodInfo ToUpper => __toUpper;
-        public static MethodInfo ToUpperInvariant => __toUpperInvariant;
-        public static MethodInfo ToUpperWithCulture => __toUpperWithCulture;
-        public static MethodInfo Trim => __trim;
-        public static MethodInfo TrimEnd => __trimEnd;
-        public static MethodInfo TrimStart => __trimStart;
-        public static MethodInfo TrimWithChars => __trimWithChars;
+        public static MethodInfo ContainsMethod => __contains;
```

Comment:
I only renamed the public property for now.

If we continue with this refactoring I will rename the fields as well.